src/main/java/com/epam/university/java/core/task002/Task002Impl.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.university.java.core.task002;
+
+public class Task002Impl implements  Task002{
+
+    @Override
+    public boolean isEquals(String firstString, String secondString) {
+        if (firstString==null | secondString==null){
+            throw new IllegalArgumentException();
+        }
+
+        return firstString.equals(secondString);
+    }
+
+    @Override
+    public String left(String sourceString, int number) {
+        if (sourceString==null ||number<0) {
+            throw new IllegalArgumentException();
+        }
+        if (number>sourceString.length()){
+            number=sourceString.length();
+        }
+
+        return sourceString.substring(0,number);
+    }
+
+    @Override
+    public String right(String sourceString, int number) {
+        if (sourceString==null ||number<0) {
```

Comment:
You should add additional spaces around operations like ==, >, <, ex: (some == any), it improves readability of code.